frontend/src/features/admin-form/create/workflow/components/WorkflowContent/EditStepBlock/RespondentBlock.tsx

```diff
@@ -44,12 +58,54 @@ export const RespondentBlock = ({
       py="1.5rem"
       px={{ base: '1.5rem', md: '2rem' }}
     >
-      <Text textStyle="subhead-3">Respondent in this step</Text>
-
       {isFirstStep ? (
-        <Text>Anyone you share the form link with</Text>
+        <>
+          <Flex alignItems="center" gap="0.5rem">
+            <Text textStyle="subhead-3">Respondent in this step</Text>
+            <Tooltip
+              placement="top"
+              label="Anyone you share the form link with"
+            >
+              <Icon as={BxsInfoCircleAlt} />
+            </Tooltip>
+          </Flex>
+          <FormControl isInvalid={!!errors.field} isRequired={false}>
```

Comment:
Typo here? We can omit specifying `isRequired={false}` since it already defaults to [false](https://v2.chakra-ui.com/docs/components/form-control/props#:~:text=false-,isRequired,-Description).